### PR TITLE
feat: add tender proposal

### PR DESCRIPTION
**Pull request by andyesp**
(Change base branch to feat/bidding-and-tendering after merging #910)

image

image

image

image

### CODE REVIEW

File: src/components/Proposal/View/ProposalVoting/GetInvolvedQuestion.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 
 import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
 import useFormatMessage, { useIntl } from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { kebabCase } from 'lodash'
```
Comment (ncomerci) [inline src/components/Proposal/View/ProposalVoting/GetInvolvedQuestion.tsx:5]: Please, change the import to:

```suggestion
import kebabCase from 'lodash/kebabCase'
```

File: src/entities/Proposal/types.ts [hunk 2]
```diff
@@ -475,6 +477,74 @@ export const newProposalPitchScheme = {
...
+      minLength: 20,
+      maxLength: 3500,
+    },
+    deliverables: {
+      type: 'string',
+      minLength: 20,
+      maxLength: 3500,
+    },
+    target_release_date: {
+      type: 'string', // TODO: Check date type
```
Comment (ncomerci) [inline src/entities/Proposal/types.ts:541]: Reminder of this

File: src/entities/Proposal/types.ts [hunk 3]
```diff
@@ -475,6 +477,74 @@ export const newProposalPitchScheme = {
...
+  type: 'object',
+  additionalProperties: false,
+  required: [
+    'linked_proposal_id',
+    'project_name',
+    'summary',
+    'problem_statement',
+    'technical_specification',
+    'use_cases',
+    'deliverables',
```
Comment (1emu) [inline src/entities/Proposal/types.ts:502]: We might be missing target release date here, I'm not sure if it's required 👁️

File: src/hooks/useURLSearchParams.ts
Comment (1emu) [inline src/hooks/useURLSearchParams.ts:1]: 🏅

File: src/pages/submit/tender.tsx [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+  deliverables: '',
+  target_release_date: new Date(),
+}
+const schema = newProposalTenderScheme.properties
+
+export default function SubmitTenderProposal() {
+  const t = useFormatMessage()
+  const params = useURLSearchParams()
+  const [account, accountState] = useAuthContext()
+  const accountBalance = isEthereumAddress(params.get('address') || '') ? params.get('address') : account
```
Comment (1emu) [inline src/pages/submit/tender.tsx:52]: I don't know why we are calling this accountBalance jaja 🙈. 
Can you also rename it in draft and governance submit pages please?

**Review by 1emu — APPROVED** [review]
🚢 🇮🇹

File: src/components/Search/SearchInputMobile.tsx [hunk 5]
```diff
@@ -6,15 +6,15 @@ import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
...
+import { useProposalsSearchParams } from '../../hooks/useSearchParams'
 
 import { handleSearch } from './SearchInput'
 import './SearchInputMobile.css'
 
 export default function SearchInputMobile(props: React.HTMLAttributes<HTMLDivElement>) {
   const t = useFormatMessage()
   const location = useLocation()
-  const { search, searching } = useSearchParams()
+  const { search, searching } = useProposalsSearchParams()
```
[the remark this answers is not in the document]
Comment (andyesp) [reply src/components/Search/SearchInputMobile.tsx:17]: I found it useful as this is only used in the /proposals page.
Created a new hook called "useURLSearchParams" for a more generic use of the search params.

File: src/entities/Proposal/types.ts (hunk 3 above)
Comment (andyesp) [reply src/entities/Proposal/types.ts:502]: You are right!

File: src/pages/submit/tender.tsx (hunk 4 above)
Comment (andyesp) [reply src/pages/submit/tender.tsx:53]: Awful copy-pasting 🤣  I think the variable is not necessary, I'll check

**Review by ncomerci — APPROVED** [review]

**Pull request merged**